scripts/generate_psa_key_attributes.py

```diff
@@ -111,22 +133,73 @@ def pack(self):
             0,  # Reserved, only used if key id encodes owner id
         )
 
+
 def is_valid_hexa_code(string):
     try:
         int(string, 16)
         return True
     except ValueError:
         return False
 
-def main() -> None:
+
+def generate_attr_file(
+    attributes: PlatformKeyAttributes,
+    key_value: str,
+    key_file: str,
+    trng_key: bool,
+    key_from_file: str,
+    bin_out: bool,
+) -> None:
+    metadata_str = binascii.hexlify(attributes.pack()).decode("utf-8").upper()
+
+    if key_file and Path(key_file).is_file():
+        with open(key_file, encoding="utf-8") as file:
+            json_data = json.load(file)
+    else:
+        json_data = json.loads('{ "version": 0, "keyslots": [ ]}')
+
+    if trng_key:
+        value = f"TRNG:{int(math.ceil(attributes.size / 8))}"
+    elif key_value:
+        key = key_value.removeprefix("0x")
+        if not is_valid_hexa_code(key):
+            print("Invalid KEY value")
+            return
+        value = f"0x{key}"
+    elif key_from_file != "":
```

Comment:
The ncs_provision script also have support PEM files, I think that support should should be removed and then the support in this file should be used so we do not end up with two different implementations for how to do support .PEM files.

Reply:
I moved implementation from ncs_provision.py to generate_psa_key_attributes.py to allow using file with private key to get public key from it. So now only the one implementation is used.